dev-project/entrypoint.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 # Add Mongodb ppa
 apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 4B7C549A058F8B6B
-echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.2 multiverse" | tee /etc/apt/sources.list.d/mongodb.list
+echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.4 multiverse" | tee /etc/apt/sources.list.d/mongodb.list
```

Comment:
We can't change this, the gpg key above is linked to this particular version

Reply:
Probably that's the reason for missing package.

It would be cleaner to upgrade mongo version. If it's not possible, we can port changes from Dockerfile

Reply:
Let's upgrade to 4.4, can you replace line 6 with `wget -qO - https://www.mongodb.org/static/pgp/server-4.4.asc | apt-key add -`

Reply:
line 8 also needs update to remove `4.2.7`

Reply:
Actually, this is cleaner than Dockerfile, I think it would be reasonable to port it to the main Dockerfile as well

Reply:
yeah